lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,69 @@
+# Gerrit
+
+## Authentication
+
+First, ![HTTP access token](/assets/images/gerrit-http-password.png) for the Renovate account.
```

Comment:
```suggestion
<figure markdown>
  ![Gerrit HTTP access token](/assets/images/gerrit-http-password.png){ loading=lazy }
  <figcaption>First, create a HTTP access token for the Renovate account.</figcaption>
</figure>
```

I think this is the correct syntax to get the image into Material for MkDocs with the text underneath the image. I'm also enabling lazy loading for the image.

Links to the Material for MkDocs manual:

- https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions
- https://squidfunk.github.io/mkdocs-material/reference/images/#image-lazy-loading

Reply:
ok, i've tried to "preview" this (`docker run --rm -it -p 8000:8000 -v ${PWD}/tmp:/docs squidfunk/mkdocs-material`), 
but there was no `mkdocs.yml` and have to write my own...

```yaml
site_name: Renovate
theme:
  name: material
markdown_extensions:
  - md_in_html
```

The image looks okay, but the lazy-loading was printed below the image....

What is the right approach for testing these generated markdown files locally?

Reply:
With this config the lazy-loading works fine.

```yaml
site_name: Renovate
theme:
  name: material
markdown_extensions:
  - md_in_html
  - admonition
  - pymdownx.details
  - pymdownx.superfences
  - attr_list
```

Reply:
We're building the docs in with MkDocs + Material for MkDocs over on the https://github.com/renovatebot/renovatebot.github.io repository.

That's why there are no build steps in the main repository. 😉